### PR TITLE
Proposal naming refactors

### PR DESCRIPTION
**Pull request by smarshall-spitzbart**
Changes all instances, comments, methods, etc. such that:

- ```CreateConsumerChainProposal``` (the abstract type) is now ```ConsumerAdditionProposal```
- ```StopConsumerChainProposal``` (the abstract type) is now ```ConsumerRemovalProposal```
- ```StopConsumerChainProposal()``` (the method) is now ```HandleConsumerRemovalProposal()```
- ```CreateConsumerChainProposal()``` (the method) is now ```HandleConsumerAdditionProposal()```

Related to #331

### CODE REVIEW

**Review by danwt — APPROVED** [review]
Amazing, I like the changes a lot. You even got in and changed the subtest name strings. Thanks for this.

File: x/ccv/provider/client/proposal_handler.go [hunk 1]
```diff
@@ -21,11 +21,11 @@ import (
...
-var ProposalHandler = govclient.NewProposalHandler(NewCreateConsumerChainProposalTxCmd, ProposalRESTHandler)
+var ProposalHandler = govclient.NewProposalHandler(NewConsumerAdditionProposalTxCmd, ProposalRESTHandler)
 
-// NewCreateConsumerChainProposalTxCmd returns a CLI command handler for creating
+// NewConsumerAdditionProposalTxCmd returns a CLI command handler for creating
 // a new consumer chain proposal governance transaction.
-func NewCreateConsumerChainProposalTxCmd() *cobra.Command {
+func NewConsumerAdditionProposalTxCmd() *cobra.Command {
 	return &cobra.Command{
 		Use:   "create-consumer-chain [proposal-file]",
```
Comment (jtremback) [inline x/ccv/provider/client/proposal_handler.go:30]: This needs to be renamed as well.

Comment (jtremback) [inline x/ccv/provider/client/proposal_handler.go:30]: This needs to be changed as well

**Review by jtremback — CHANGES_REQUESTED** [review]
Proposals are still created on the CLI using `create-consumer-chain`

**Comment by smarshall-spitzbart** [comment]
> Proposals are still created on the CLI using `create-consumer-chain`

Great catch, the cli command should now look like 
```bash
tx gov submit-proposal consumer-addition <path/to/proposal.json> --from=<key_or_address>
```

Changes in https://github.com/cosmos/interchain-security/pull/354/commits/15d8abc47439d4690251f65d815efb95a6d02bec

**Review by sainoe — APPROVED** [review]
Clean renaming good work 👍

**Review by jtremback — APPROVED** [review]

**Pull request merged**